data-model/SDP-sample-records/textual-artifact_sample1.json

```diff
@@ -0,0 +1,65 @@
+{
+  "id": "ark:/21198/adf20c1f",
+  "type": "Textual Artifact",
+  "label": "Sinai Syriac 2A, ff. 1-152v, Gospels",
+  "unit_start": {
+    "parent_id": "ark:/21198/z1x07bdf",
+    "locus": "1"
+  },
+  "unit_end": {
+    "parent_id": "ark:/21198/z1x07bdf",
+    "locus": "152"
+  },
+  "work": "https://sinaimanuscripts/library.ucla.edu/work/1",
+  "associated_persons": [
+    {
+      "id": "https://sinaimanuscripts/library.ucla.edu/person/42",
+      "attested_name_NS": "ܕܩܕܝܫܐ ܡܪܝ ܝܘܚܢܢ",
+      "attested_name": "Qadisho Mar Yuhanon",
+      "role": "attributed-author",
+      "source": "Final rubric on f. 152v"}
+  ],
+  "rubric": {},
+  "incipit": {},
+  "quotes": [],
+  "explicit": {
+    "locus": "f. 152v",
+    "transcription": {"syr": ["ܐܝܬ ܕܝܢ ܐܦ ܐܚܪ̈ܢܝܬܐ ܣܓ̈ܝܬܐ ܕܥܒܕ ܝܫܘܥ ܐܝܠܝܢ ܕܐܠܘ ܚܕܐ ܚܕܐ ܡܬܟܬܒܢ ܗܘ̈ܝ. ܐܦ ܠܐ  ܗܘ ܥܠܡܐ ܐܝܟ ܕܣܒܪ ܐܢܐ. ܣܦܩ ܗܘܐ ܠܟܬܒ̈ܐ ܕܡܬܟܬ̈ܒܝܢ "]},
+    "translation": {"en": ["There were also many other things which Jesus did that, were each one written, I believe, not even the world would be sufficient for the books which would be written."]}
+  },
+  "final_rubric": {
+    "locus": "f. 152v",
+    "transcription": {"syr": ["ܫܠܡ ܐܘܢܓܠܝܘܢ ܕܩܕܝܫܐ ܡܪܝ ܝܘܚܢܢ ܕܡܠܠ ܝܘܢܐܝܬ ܒܐܦܣܘܣ"]},
+    "translation": {"en": ["Finished is the Gospel of Saint John which he spoke in Greek in Ephesus."]}
+  },
+  "defective": true,
+  "condition_note": "Incomplete at the beginning. Syriac 2A originally belonged together with ms. Chabot 12 (Milan, Biblioteca Ambrosiana).",
+  "language": "Syriac",
+  "toc": ["Matthew, f. 1r-32v (incomplete at the beginning)", "Mark, f. 32v-70r", "Luke, f. 70r-105v", "John, f. 106r-152v"],
```

Comment:
we still haven't figured out relationships between manuscripts' textual artifacts. So this is an area to discuss how we use a TOC to show hierarchies, etc. And we will also have to work out what constitutes a textual artifact -- i.e., what gets a record? and what remains just in 'contents notes'

Reply:
Add a contents note field

Reply:
contents note added. also re-did contents to have 'note', 'features', and 'toc' fields. I made the 'toc' field a bit more elaborate, as we discussed, so we could capture things like notes, features, and extent. Could also move to include data about paratexts, etc. in the toc field.